calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
+
+exports.create_calculator = function (operator) {
+  const numbers = [];
+
+  return {
+    add_number: function (number) {
```

Comment:
Rather ambiguous to use "add" in this way, when later in the same file we use "add" in a different way. 
(adding item to list, as opposed to adding numbers together)

Recommend renaming to avoid this confusion. Perhaps "append_number_to_inputs"?